### DISCUSSION TITLE
Manually place cameras

### DISCUSSION THREAD

**Discussion by vintheruler1** [alicevision/Meshroom#2061] [Q&A] [answered]
So I was looking in at the source code, and I was wondering where the algorithm to place the cameras based on its focal length and order is done? I've searched the `StructureFromMotion` but all I see is calling classes in the desc.py file.

**Comment by natowi** [comment, marked as answer]
If you are looking for the SfM algorithm you need to look in the AliceVision repository: https://github.com/alicevision/AliceVision/tree/develop/src/aliceVision/sfm
